provd/go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/canonical/ubuntu-desktop-provision/provd
 
-go 1.22.0
+go 1.21.7
```

Comment:
It downgrades go?

Reply:
Yea, I ran into issues there on the runners, let me give it another look and see if I can fix it :slightly_smiling_face: